data-sets-api-server/src/test/java/org/zowe/data/sets/services/zosmf/ZosmfDataSetServiceTest.java

```diff
@@ -77,10 +74,10 @@ public void testListDataSetMembersRunnerExceptionThrown() throws Exception {
     public void testListDataSetsRunnerValueCorrectlyReturned() throws Exception {
         String filter = "DATA.SET.NAME*";
 
-        DataSetAttributes att1 = DataSetAttributes.builder().name("dataset1").build();
-        DataSetAttributes att2 = DataSetAttributes.builder().name("dataset2").build();
+        DataSet att1 = DataSet.builder().name("dataset1").build();
+        DataSet att2 = DataSet.builder().name("dataset2").build();
 
-        List<DataSetAttributes> expected = Arrays.asList(att1, att2);
+        List<DataSet> expected = Arrays.asList(att1, att2);
```

Comment:
Please create a new test for list dataset name and exceptions, rather than replacing an existing one